kyo-core/shared/src/test/scala/kyo/ClockTest.scala

```diff
@@ -425,7 +425,7 @@ class ClockTest extends Test:
                     _        <- task.interrupt
                     instants <- queue.drain
                 yield
-                    intervals(instants).foreach(v => assert(v == 1.millis))
+                    intervals(instants).foreach(v => assert(v <= 2.millis))
```

Comment:
SN has more drift in thread wake ups. That might be the cause of the issue with the scheduler regulator in SN. A different tuning might be enough cc/ @hearnadam